nilearn/plotting/displays.py

```diff
@@ -938,6 +938,22 @@ def add_markers(self, marker_coords, marker_color='r', marker_size=30,
             marker_coords_2d, third_d = _coords_3d_to_2d(
                 marker_coords, direction, return_direction=True)
             xdata, ydata = marker_coords_2d.T
+	        # Allow markers only in their respective hemisphere when appropriate
```

Comment:
Here you break the identation, leading to errors in the code. Can you fix it ?

Reply:
To make sure that this is right, include a pep8.flake8 checker in you Python editor.

Reply:
Yes, of course, sorry about this, I didn't notice